app/models/friend_request.rb

```diff
@@ -2,4 +2,97 @@ class FriendRequest < ApplicationRecord
 
   belongs_to :sender, class_name: "User"
   belongs_to :receiver, class_name: "User"
+
+  def self.current_user_sent_friend_requests(user)
+    user.sent_friend_requests.map do |request|
+      {
+        id: request.id,
+        receiver_id: request.receiver.id,
+        full_name: request.receiver.full_name
+      }
+    end
+  end
+
+  def self.current_user_received_friend_requests(user)
+    user.received_friend_requests.map do |request|
+      {
+        request_id: request.id,
+        sender_id: request.sender.id,
+        full_name: request.sender.full_name
+      }
+    end
+  end
```

Comment:
You are preparing a response, right? I think it's better to use serializer

Reply:
Added serializer.